packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -1383,7 +1383,7 @@ void main() {
 
     await tester.pump();
 
-    // Both Heros exist and seated in their normal parents.
+    // Both Heroes exist and seated in their normal parents.
```

Comment:
... and are seated in their normal parents.

I'm not sure we use "seated" anywhere else. I think we mean that the heroes haven't been reparented to the overlay yet, i.e. their flights haven't started.